maven-enforcer-plugin/src/it/projects/ban-dynamic-versions/pom.xml

```diff
@@ -109,6 +112,12 @@
       <!--  version range syntax with equal upper and lower bounds -->
       <version>[1.0]</version>
     </dependency>
+    <dependency>
+      <!-- banned transitive version range, but ignored -->
+      <groupId>org.apache.maven.plugins.enforcer.its</groupId>
+      <artifactId>menforcer480</artifactId>
```

Comment:
new test artifact? I don't see it in PR ...

Reply:
Ah, I misunderstood this setup. I now see the `repository` dir. Will reuse of the of existing ones and push a commit in a bit.

Reply:
If any existing artifact meet your requirement you can reuse, if no please create new

Reply:
Rebased and added a commit :)